### PR TITLE
Upgrading elixir version to 1.11.3

### PR DESCRIPTION
**Pull request by rkledesma**
[Story](https://rentpath.atlassian.net/browse/SRV-2802)

### CODE REVIEW

**Review by pasha — APPROVED** [review]
LGTM 👍

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 FROM erlang:23.2.3-alpine
 
 # elixir expects utf8.
-ENV ELIXIR_VERSION="v1.11.2" \
-    ELIXIR_VERSION_SHA="318f0a6cb372186b0cf45d2e9c9889b4c9e941643fd67ca0ab1ec32710ab6bf5" \
+ENV ELIXIR_VERSION="v1.11.3" \
+    ELIXIR_VERSION_SHA="d961305e893f4fe1a177fa00233762c34598bc62ff88b32dcee8af27e36f0564" \
```
Comment (michaelstalker) [inline Dockerfile:5]: Where did you find the SHA? José Valim hadn't posted it on the GitHub Release.

Comment (pasha) [reply Dockerfile:5]: i was able to get this by using `openssl`
```shell
~/Downloads $ openssl sha256 < elixir-1.11.3.tar.gz
d961305e893f4fe1a177fa00233762c34598bc62ff88b32dcee8af27e36f0564
```

Comment (KevinMarth) [reply Dockerfile:5]: Fwiw, this is also what `brew` has computed. https://github.com/Homebrew/homebrew-core/blob/master/Formula/elixir.rb. The lack of attention to this on the Elixir site is disappointing.

**Review by KevinMarth — APPROVED** [review]

**Review by Stratus3D — APPROVED** [review]

**Pull request merged**